src/navigation/underline-nav.scss

```diff
@@ -29,6 +29,8 @@
   &:focus {
     text-decoration: none;
     border-bottom-color: $border-gray-dark;
+    outline: 1px dotted transparent; // Support Firfox custom colors
```

Comment:
@simurai doesn't adding `outline-color: transparent` to the `:focus` state break accessibility in all other browsers or when users don't use custom colors?